.travis.yml

```diff
@@ -82,6 +82,7 @@ env:
    - C=mor1kx           TC="ise"       P=opsis             T="base net"
    - C=mor1kx           TC="ise"       P=pipistrello       T="base"
    # VexRISCV
+   - C=vexriscv         TC="vivado"    P=netv2             T="base"
```

Comment:
NeTV2 should probably have a net target as well?

Reply:
In theory yes, but at this point NeTV2 lacks a `net.py` file that instantiates the network interface. I focused more on PCIe side of things in this PR and I didn't test the network interface

Reply:
I added a network target for NeTV2 and modified the `.travis.yml` to test it.